index.html

```diff
@@ -70,5 +70,198 @@ <h2 class="first-title">I'm Peter <br/> Glad to see you! </h2>
 </div>
 </div>
 </section>
+<section class="card-works" id="portfolio">
+    <div class="card card-works-layout1">
+        <a href ="#">
+            <img src="/Images/SnapshootPortfolio1.png" alt="SnapshootPortfolio1" class="snapshot1">
```

Comment:
[OPTIONAL]

- [ ] It's recommended that you use a `relative path` when you're linking an image or a file in your HTML, for example in this case the `src` path should be like this: `src="./Images/SnapshootPortfolio1.png" `, the difference between the normal path and relative path is that it locates the file or folder starting from the root of the file system. 😁 Please do this for all the paths you have.